AIPscan/Data/report_data.py

```diff
@@ -602,6 +602,7 @@ def storage_locations(storage_service_id, start_date, end_date):
         loc_info[fields.FIELD_STORAGE_LOCATION] = location.description
         loc_info[fields.FIELD_AIPS] = location.aip_count(start_date, end_date)
         loc_info[fields.FIELD_SIZE] = location.aip_total_size(start_date, end_date)
+        loc_info[fields.FIELD_FILE_COUNT] = location.file_count(start_date, end_date)
```

Comment:
Nitpick: the comments for this method have incorrect :param and :returns: values.